### PR TITLE
Add docker compose & logging services

### PR DESCRIPTION
**Pull request by rashboldb**
### 🛠 Changes being made
This resolves issue https://github.com/COXIT-CO/RecruitFlowAI/issues/32

* Added docker compose 
* Added Grafana, Loki & Promtail services for comfort logging 
* Removed layer from Dockerfile where we copy .env, env provided via docker compose now

**Known issues:** 
This is not the best practice to expose direct access to the Docker daemon. Make sure you won't use malicious software alongside/inside the provided containers or use a more sophisticated approach for logs sharing.   

### 🧪 How was it tested? / Demo
Tested manually, after the services started visit localhost at port 3200 to access Grafana and enjoy your logs
### 📸 Screenshots

image

### 🏎 Quality check

- [ ]  Set base branch to main.
- [x]  Pull request is prepared: Added to sprint, added yourself as assignee, reviewers are added.
- [X]  PR is not very big (about 200 lines of change ideally).
- [x]  I have added comments to my code, particularly in hard-to-understand areas.
- [x]  Doesn't create any new FIXME or TODO comments. If not, please explain.
- [x]  Tested manually (explained the scenario, added screenshots)
- [x]  I have tested my changes locally before pushing them to this branch.
- [x]  I have updated the documentation accordingly.
- [x]  Followed best practice rules:https://google.github.io/styleguide/pyguide.html

### CODE REVIEW

File: logging_configs/grafana-datasources.yml
Comment (iramykytyn) [inline logging_configs/grafana-datasources.yml:1]: move the folder with scripts to `./scripts/loki` folder

**Review by iramykytyn — CHANGES_REQUESTED** [review]
Update readme how to start bot now with docker compose.

**Review by iramykytyn — COMMENTED** [review]
Could you record loom how to setup grafana so it's clear how to record logs though public access to IP? Now I can see grafana UI but I can't login https://www.loom.com/share/f7c0f1bd19ca4bcd9267974dddc0b98e.

It will be helpful to have that loom attached to readme with info how to view logs.

File: docker-compose.yml
Comment (iramykytyn) [inline docker-compose.yml:1]: probably configure some services to depend on others. For example to not start bot if Loki is not up yet. But think about it carefully if it makes sense to you.

Comment (rashboldb) [reply docker-compose.yml:1]: not sure if we want to have this. I can imagine scenarios where we'd like to run bot and not to run Loki and vice versa. the only necessary dependency here is bot being dependent on cloudflare tunnel container. which we don't have in this PR yet, so I'll add this to the https://github.com/COXIT-CO/RecruitFlowAI/pull/36

**Comment by iramykytyn** [comment]
https://www.loom.com/share/591ebf77515d49ea82a13ed4aa605c6b

**Comment by rashboldb** [comment]
> https://www.loom.com/share/591ebf77515d49ea82a13ed4aa605c6b

Here I demonstrate that the logs being sent and we can access the fresh ones. 
https://www.loom.com/share/d43e8b32eec8448d9c16d955ebdd66c6?sid=80a47f21-a801-4632-91dc-778615426091

It's not obvious rn, what's the reasons behind the failure in your case. Have to investigate more on the quesion, but it'll be hard to debug without being able to reproduce.

**Review by iramykytyn — APPROVED** [review]
looks good,  only fix conflicts

**Pull request merged**